backend/api_handler.py

```diff
@@ -32,6 +33,11 @@
 for name in ['boto3', 's3transfer', 'botocore', 'boto']:
     logging.getLogger(name).setLevel(logging.ERROR)
 
+ALLOW_INTROSPECTION = True if os.getenv('ALLOW_INTROSPECTION') == 'True' else False
+
+if not ALLOW_INTROSPECTION:
+    did_you_mean.__globals__['MAX_LENGTH'] = 0
```

Comment:
Just curious, what is this doing? It is using https://graphql-core-3.readthedocs.io/en/latest/modules/pyutils.html#graphql.pyutils.did_you_mean, but what is it accessing?

Reply:
I guess this is to disable suggestions returned by the GQL server to avoid exposing fields.